tools/automation/cache/buildNewCache.sh

```diff
@@ -0,0 +1,12 @@
+bundle install
```

Comment:
Small nit pick, I think the file naming convention is to stick with snake case 🐍